autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -18,6 +18,7 @@
   <include file="$(find-pkg-share vehicle_launch)/launch/vehicle.launch.xml">
     <arg name="vehicle_model" value="$(var vehicle_model)"/>
     <arg name="sensor_model" value="$(var sensor_model)"/>
+    <arg name="vehicle_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.yaml"/>
```

Comment:
`vehicle_info.yaml` will be renamed at https://github.com/tier4/lexus_description.iv.universe/pull/11. cc @kenji-miyake

Reply:
I can create another PR after this is merged!